### ISSUE TITLE
Block Editor: Accessibility issue editing links

### ISSUE THREAD

**Issue by melindahelt** [wordpress-mobile/gutenberg-mobile#2360]
Accessibility issue reported by a user:

#2937683-zen

hello, I have a problem on WordPress iOS app; I am a blind user, and I have set my iphone with its accessibility feature. Settings -> accessibility -> voiceover. Well, I have encountered a bug in your app, envolving VoiceOver usage; it occurs when you have placed links inside an article and have saved it into a draft.
Step-by-step:
1. Write an article including a link, using the "paragraph" block. The draft may be created using desktop platform.
2. Save the draft.
3. Open it with iOS app, and VoiceOver on.
4. Place VoiceOver focus on the block you wish to modify, where the link is.
5. Double-tap on the block...

The app will open the link, youtube or safari whatever it opens... And does not allow you to access keyboard to modify article text.
Only way to access editing function, is to press firmly on the screen using 3D touch, but soon 3D touch will disappear from iPhones...
Now I have no workaround for this, and I must remember to place a special marker in the words I want to link, then write it all, and place links as last element before publishing, on desktop. Please let me know!
Links should be opened only if I decide to do it, selecting "link" item from VoiceOver rotor.

##### Tested on [device], iOS [version], WPiOS [version]
iPhone X, iOS 13.4.5, WPiOS 14.6

**Comment by jd-alexander** [comment]
Hi @etoledom I was just looking at this issue right now. So far the ideas I have based on the issue, would be for us to first determine if the device is in accessibility mode. Once that's done we would then disable to the link opening ability of the paragraph block and then utilize a [custom rotor on iOS ](https://developer.apple.com/documentation/uikit/uiaccessibilitycustomrotor) so show the options of either opening or editing a link. In terms of having multiple links in a single parapraph I stumbled [on an accessibility article](https://callstack.com/blog/react-native-android-accessibility-tips/) by @lukewalczak . I am not sure if any of the approaches described in the article can be applicable to this, as I am still exploring how the editor works. Nonetheless, I am just sharing these thoughts here in case you have anything to add :)

**Comment by etoledom** [comment]
Hey @jd-alexander !

I do have something to add, not sure if it might be related:
We had problems tapping links on Paragraph blocks before (normal taps, not related to VoiceOver).

You can see t[his that has some information](https://github.com/WordPress/gutenberg/blob/6e379199d64eca47f8c2b9d28a6f40c3ad0c456a/packages/react-native-aztec/ios/RNTAztecView/RCTAztecView.swift#L166).

Basically the UIKit API wasn't behaving as the documentation described, so we disabled it.

This might be related in a way that we should also disabled taping on links for VoiceOver interactions. But I'm not sure if VoiceOver is affected by the same UIKit bug.
